scripts/_runCommand.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+const chalk = require('chalk');
+const spawnSync = require('child_process').spawnSync;
+
+/**
+ * @example `runCommands('npm test', packageDirectory)`
+ * @example `runCommands(['npm test', 'npm publish'], packageDirectory)`
+ * @param {boolean} options.suppressOutput don't print output to stdout
+ */
```

Comment:
Ah. I see. But it seems as the scripts-files aren't run in a flow context? Maybe this is a separate issue / bigger rewrite to be able to run scripts as these in flow?